docs/migration-guide/to-15.md

```diff
@@ -11,7 +11,84 @@ Two significant changes may affect you:
 
 ### Deprecated stylistic rules
 
-We've deprecated [76 of the rules that enforce stylistic conventions](../user-guide/rules.md#deprecated), e.g. `indentation`.
+We've deprecated 76 of the rules that enforce stylistic conventions:
+
+- `at-rule-name-case`
+- `at-rule-name-newline-after`
+- `at-rule-name-space-after`
+- `at-rule-semicolon-newline-after`
+- `at-rule-semicolon-space-before`
+- `block-closing-brace-empty-line-before`
+- `block-closing-brace-newline-after`
+- `block-closing-brace-newline-before`
+- `block-closing-brace-space-after`
+- `block-closing-brace-space-before`
+- `block-opening-brace-newline-after`
+- `block-opening-brace-newline-before`
+- `block-opening-brace-space-after`
+- `block-opening-brace-space-before`
+- `color-hex-case`
+- `declaration-bang-space-after`
+- `declaration-bang-space-before`
+- `declaration-block-semicolon-newline-after`
+- `declaration-block-semicolon-newline-before`
+- `declaration-block-semicolon-space-after`
+- `declaration-block-semicolon-space-before`
+- `declaration-block-trailing-semicolon`
+- `declaration-colon-newline-after`
+- `declaration-colon-space-after`
+- `declaration-colon-space-before`
+- `function-comma-newline-after`
+- `function-comma-newline-before`
+- `function-comma-space-after`
+- `function-comma-space-before`
+- `function-max-empty-lines`
+- `function-parentheses-newline-inside`
+- `function-parentheses-space-inside`
+- `function-whitespace-after`
+- `indentation`
+- `linebreaks`
+- `max-empty-lines`
+- `max-line-length`
+- `media-feature-colon-space-after`
+- `media-feature-colon-space-before`
+- `media-feature-name-case`
+- `media-feature-parentheses-space-inside`
+- `media-feature-range-operator-space-after`
+- `media-feature-range-operator-space-before`
+- `media-query-list-comma-newline-after`
+- `media-query-list-comma-newline-before`
+- `media-query-list-comma-space-after`
+- `media-query-list-comma-space-before`
+- `no-empty-first-line`
+- `no-eol-whitespace`
+- `no-extra-semicolons`
+- `no-missing-end-of-source-newline`
+- `number-leading-zero`
+- `number-no-trailing-zeros`
+- `property-case`
+- `selector-attribute-brackets-space-inside`
+- `selector-attribute-operator-space-after`
+- `selector-attribute-operator-space-before`
+- `selector-combinator-space-after`
+- `selector-combinator-space-before`
+- `selector-descendant-combinator-no-non-space`
+- `selector-list-comma-newline-after`
+- `selector-list-comma-newline-before`
+- `selector-list-comma-space-after`
+- `selector-list-comma-space-before`
+- `selector-max-empty-lines`
+- `selector-pseudo-class-case`
+- `selector-pseudo-class-parentheses-space-inside`
+- `selector-pseudo-element-case`
+- `string-quotes`
+- `unicode-bom`
+- `unit-case`
+- `value-list-comma-newline-after`
+- `value-list-comma-newline-before`
+- `value-list-comma-space-after`
+- `value-list-comma-space-before`
+- `value-list-max-empty-lines`
```

Comment:
Maybe add a sentence with a link to https://stylelint.io/user-guide/cli/#--quiet-deprecation-warnings?

Reply:
Since the flag was only introduced past `15.0.0`, would it make sense to add a sentence in the `to-15` guide? And/or any suggestions on how to phrase the sentence?

Reply:
People would benefit from knowing about that option now, less so when we release 16.0.0. I'll open a PR to add it.